DailyReportWeb/Controllers/ReportController.cs

```diff
@@ -28,10 +36,53 @@ public ActionResult ResendDraft(string id, DateTime date)
             if (date.Date != DateTime.Today)
                 return Content("Cannot resend draft for another date");
 
-            if (ReportRunner.TryRunReport(id + "draft"))
+            if (ReportRunner.TryRunReport(id))
                 return Content("Draft report was resent");
-            else 
+            else
                 return Content("Error in resending draft report");
-        }     
+        }
+
+        [HttpGet]
+        public ActionResult SendIndividualDraft(string id, DateTime date, string draftKey)
+        {
+            if (date.Date != DateTime.Today)
+                return Content("Cannot confirm report for another date");
+
+            var policyPath = PolicyService.GetPolicyPath(id);
+            var policy = Policy.CreateFromFile(policyPath);
+
+            var confirm = ReportRunner.SetIndividualDraftConfirmation(draftKey, policy, policyPath);
+            if (!confirm)
+                return Content("Error in confirmation");
+
+            if (ReportRunner.IndividualReportConfirmedByAll(policy))
+            {
+                PolicyService.SetPolicyFullDraft(policy, policyPath);
+
+                if (ReportRunner.TryRunReport(id) == true)
+                    return Content("Report confirmed. Full draft sent");
+                else
+                    return Content("Report confirmed. Error in sending full draft report");
+            }
+
+            return Content("Report confirmed");
+        }
+
+        [HttpGet]
+        public ActionResult ResendIndividualDraft(string id, DateTime date, string draftKey)
+        {
+            if (date.Date != DateTime.Today)
+                return Content("Cannot resend report for another date");
+
+            var policy = PolicyService.LoadPolicy(id);
+
+            if (!ReportRunner.CheckConfirmed(draftKey, policy))
```

Comment:
Aici ar trebui sa fie invers. 
sa faci If( ...) return Content ("Draft already sent"); si apoi in afara if-ului sa pui cazul de baza.